cms/templates/visibility_editor.html

```diff
@@ -31,108 +35,78 @@ <h4 class="title">${_('No content groups exist')}</h4>
             <span class="icon fa fa-exclamation-triangle" aria-hidden="true"></span>
             <p class="copy">
                 ## Translators: Any text between {screen_reader_start} and {screen_reader_end} is only read by screen readers and never shown in the browser.
-                ${_(
-                    "{screen_reader_start}Warning:{screen_reader_end} The Unit this component is contained in is hidden from students. Visibility settings here will be trumped by this."
-                    ).format(
-                        screen_reader_start='<span class="sr">',
-                        screen_reader_end='</span>',
+                ${Text(_(
+                    "{screen_reader_start}Warning:{screen_reader_end} The unit that contains this component is hidden from learners. The unit setting overrides the component visibility settings defined here."
+                    )).format(
+                        screen_reader_start=HTML('<span class="sr">'),
+                        screen_reader_end=HTML('</span>'),
                     )
                 }
             </p>
         </div>
     % endif
 </div>
 
-% if len(user_partitions) > 0:
+% if len(selectable_partitions) > 0:
     <form class="visibility-controls-form" method="post" action="">
-
+        <div role="group" aria-labelledby="visibility-title">
```

Comment:
@cahrens Yeah, I don't think you want that first role=group since the current state (Access is not restricted) does not describe the settings below.  They seem independent to me.  You have a role=group on the related form controls, which makes sense, but I don't think you need this one.